project/tests/invalidPassword.test.js

```diff
@@ -0,0 +1,74 @@
+const { test, expect } = require('@playwright/test');
+const FormSteps = require('../steps/formSteps');
+const logger = require('../../logger');
+const testData = require('../testData/testData.json'); // Load test data
+
+test('Invalid password', async ({ page }) => {
```

Comment:
Invalid and valid password cases have same logic, move two cases in one file and create method with same logic:

test('Invalid password', async ({ page }) => {
    fillCardFields(invalidPassword)
    expect()
});

test('Invalid password', async ({ page }) => {
    fillCardFields(validPassword)
    expect()
});

async function fillCardFields(password) {}